src/app/integrations/edit-page/flow-view/flow-view.component.html

```diff
@@ -21,17 +20,27 @@
           [currentPosition]="currentPosition"
           [currentState]="currentState"></ipaas-integrations-flow-view-step >
       </div>
-      <div class="row steps" *ngFor="let step of getMiddleSteps(); let position = index;">
-        <div *ngIf="currentPosition === 'new'">
-          New</div>
+      <div class="row steps" *ngIf="currentState === 'save-or-add-step'">
```

Comment:
I love this; much, much easier to understand and easier to see right away when looking at the markup. Usually not a fan of having logic in the markup, but this is really minimal.